src/ui/hooks/useTime.ts

```diff
@@ -1,17 +1,19 @@
+import { MILLISECONDS } from '~constants';
+import { useUtils } from './useDependency';
 import { useEffect, useState } from 'react';
-import moment, { Moment } from 'moment';
 import { useOnAppFocus } from './useOnAppFocus';
 
-export const useTime = (): Moment => {
-  const [time, setTime] = useState(moment());
+export const useTime = (): number => {
```

Comment:
This was probably the most 'difficult' change here.. can test it quite easily by leaving the app on and coming back in a couple mins.. if the time matches your device/system time, it's working 😁